Sources/Stencil/AssetsCatalogContext.swift

```diff
@@ -38,11 +38,19 @@ extension AssetsCatalogParser {
       switch entry {
       case let .group(name: name, items: items):
         return [
+          "type": "group",
           "name": name,
           "items": structure(entries: items)
         ]
+      case let .color(name: name, value: value):
+        return [
+          "type": "color",
+          "name": name,
+          "value": value
```

Comment:
See my comment in the PR, we can't do that for components, colors can be different on a device basis (and other criteria).